static/app/views/issueDetails/streamline/eventNavigation.tsx

```diff
@@ -0,0 +1,371 @@
+                  size: 'zero',
+                  borderless: true,
+                  showChevron: false,
+                }}
+                position="bottom"
+                size="xs"
+                items={[
+                  {
+                    key: 'copy-event-id',
+                    label: t('Copy Event ID'),
+                    onAction: copyEventId,
+                  },
+                  {
+                    key: 'copy-event-link',
+                    label: t('Copy Event Link'),
+                    onAction: copyLink,
+                  },
+                  {
+                    key: 'view-json',
+                    label: t('View JSON'),
+                    onAction: downloadJson,
+                  },
+                ]}
+              />
+            </EventIdInfo>
+            <TimeSince date={event.dateCreated ?? event.dateReceived} css={grayText} />
+          </EventInfo>
+          <JumpTo>
+            <div>{t('Jump to:')}</div>
+            <ButtonBar>
+              {jumpToSections.map(jump => (
+                <Button
+                  key={jump.section}
+                  onClick={() => {
+                    document
+                      .getElementById(jump.section)
+                      ?.scrollIntoView({block: 'start', behavior: 'smooth'});
+                  }}
+                  borderless
+                  size="xs"
+                  css={grayText}
+                >
+                  {jump.label}
+                </Button>
+              ))}
+            </ButtonBar>
+          </JumpTo>
+        </EventInfoJumpToWrapper>
+        <Divider />
+      </div>
+    );
+  }
+);
+
+const EventNavigationWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  font-size: ${p => p.theme.fontSizeSmall};
+  padding: ${space(1)} ${space(1.5)};
+`;
+
+const NavigationWrapper = styled('div')`
+  display: flex;
+`;
+
+const Navigation = styled('div')`
+  display: flex;
+  border-right: 1px solid ${p => p.theme.gray100};
+`;
+
+const EventInfoJumpToWrapper = styled('div')`
+  display: flex;
+  gap: ${space(1)};
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: ${space(1)} ${space(2)};
+`;
+
+const EventInfo = styled('div')`
+  display: flex;
+  gap: ${space(1)};
+  flex-direction: row;
+  align-items: center;
+`;
+
+const JumpTo = styled('div')`
+  display: flex;
+  gap: ${space(1)};
+  flex-direction: row;
+  align-items: center;
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+`;
+
+const Divider = styled('hr')`
+  border-color: ${p => p.theme.border};
+  margin: 0;
+`;
+
+const EventIdInfo = styled('span')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.25)};
+`;
+
+const EventId = styled('span')`
+  position: relative;
+  font-weight: ${p => p.theme.fontWeightBold};
+  text-decoration: underline;
+  /** gray200 w/ 50% opacity */
+  text-decoration-color: rgba(128, 112, 143, 0.5);
```

Comment:
i think we generally do this with the color library at runtime. not super performant but safe for future changes